# Request
# emtpy
---
# Response
geometry_msgs/PoseStamped pose
